package.json

```diff
@@ -33,7 +33,6 @@
     "express-ws": "2.0.0-rc.1",
     "fs-promise": "0.5.0",
     "immutable": "3.7.6",
-    "lodash": "4.13.1",
```

Comment:
I wonder why removing this line didn't make the tests fail. Linting tests use `lodash/without` and `lodash/difference`